app/lib/methods/handleMediaDownload.ts

```diff
@@ -76,7 +76,7 @@ const getExtension = (type: MediaTypes, mimeType?: string) => {
 	// Audios sent by Android devices are in the audio/aac format, which cannot be converted to mp3 by iOS.
 	// However, both platforms support the m4a format, so they can maintain the same behavior.
```

Comment:
This comments is now outdated. You should reword it.

Reply:
Done